providers/apache/hive/src/airflow/providers/apache/hive/hooks/hive.py

```diff
@@ -865,7 +865,7 @@ def get_conn(self, schema: str | None = None) -> Any:
         username: str | None = None
         password: str | None = None
 
-        db = self.get_connection(self.hiveserver2_conn_id)  # type: ignore
+        db = self.get_connection(self.hiveserver2_conn_id)
```

Comment:
Had the same in vertica provider, you need to switch to
```suggestion
        db = self.get_connection(self.get_conn_id())
```